components/colorloupe/stories/template.js

```diff
@@ -1,21 +1,203 @@
+                <rect width="8" height="8" transform="translate(346 293)" fill="var(--spectrum-colorcontrol-checkerboard-dark-color)"/>
+              </g>
+              <g transform="translate(-306 -221)">
+                <rect width="8" height="8" transform="translate(338 285)" fill="var(--spectrum-colorcontrol-checkerboard-dark-color)"/>
+                <rect width="8" height="8" transform="translate(338 293)" fill="var(--spectrum-colorcontrol-checkerboard-light-color)"/>
+                <rect width="8" height="8" transform="translate(346 285)" fill="var(--spectrum-colorcontrol-checkerboard-light-color)"/>
+                <rect width="8" height="8" transform="translate(346 293)" fill="var(--spectrum-colorcontrol-checkerboard-dark-color)"/>
+              </g>
+              <g transform="translate(-290 -221)">
+                <rect width="8" height="8" transform="translate(338 285)" fill="var(--spectrum-colorcontrol-checkerboard-dark-color)"/>
+                <rect width="8" height="8" transform="translate(338 293)" fill="var(--spectrum-colorcontrol-checkerboard-light-color)"/>
+                <rect width="8" height="8" transform="translate(346 285)" fill="var(--spectrum-colorcontrol-checkerboard-light-color)"/>
+                <rect width="8" height="8" transform="translate(346 293)" fill="var(--spectrum-colorcontrol-checkerboard-dark-color)"/>
+              </g>
+            </g>
+          </g>
+
+          <clipPath id="spectrum-loupe-clip">
+            <use xlink:href="#spectrum-inner-loupe" />
+          </clipPath>
+          <clipPath id="express-loupe-clip">
+            <use xlink:href="#express-inner-loupe" />
+          </clipPath>
+          <clipPath id="express-checkerboard-loupe-clip">
+            <use xlink:href="#express-inner-loupe" transform="translate(14688 -17741)" />
+          </clipPath>
+
+          <filter id="loupe-shadow" x="-16px" y="-16px" width="96" height="128" filterUnits="userSpaceOnUse">
+            <feOffset input="SourceAlpha" />
+            <feGaussianBlur stdDeviation="2" result="d" />
+            <feFlood flood-opacity="0.302" />
+            <feComposite operator="in" in2="d" />
+            <feComposite in="SourceGraphic" />
+          </filter>
+        </defs>
+
+      ${!express ? html` <g transform="translate(4 4)" class="${rootClass}-spectrum">
+          <g clip-path="url(#spectrum-loupe-clip)">
+            <g transform="translate(-353.999 -289)">
+              <use xlink:href="#loupe-checkerboard" />
+            </g>
+          </g>
+          <g filter="url(#loupe-shadow)">
+            <use xlink:href="#spectrum-inner-loupe" />
+          </g>
+          <path class="${rootClass}-outer" d="M24,2A21.98,21.98,0,0,0,2,24c0,6.2,4,14.794,11.568,24.853A144.233,144.233,0,0,0,24,61.132,144.085,144.085,0,0,0,34.4,48.893C41.99,38.816,46,30.209,46,24A21.98,21.98,0,0,0,24,2m0-2A24,24,0,0,1,48,24c0,16.255-24,40-24,40S0,40.255,0,24A24,24,0,0,1,24,0Z"/>
+        </g>` : ''}
```

Comment:
As is, these conditionals are not working as intended. The HTML is changing based on if express is true or not but the HTML does not render.